jdaviz/configs/mosviz/helper.py

```diff
@@ -92,19 +92,26 @@ def _update_spec1d_x_axis(self, change):
             if val != old_val:
                 setattr(scales['x'], name, val)
 
-    def load_data(self, spectra_1d, spectra_2d, images=None, spectra_1d_label=None,
+    def load_data(self, directory=None, instrument=None, spectra_1d=None,
```

Comment:
Are you sure you want to break the API? This is not backward compatible.

Reply:
I don't think so, is there a better way to do this?

Reply:
The easiest way is to add new keywords at the end of the signature. That should be backward compatible but might be messy if your order also implies some sort or priority or preference.

At the very least, keep the positional arguments where they are. If you stick the new keywords to just after positional, it will break usage of those who use keyword as positional, but not everything. You can also add a `*` to disallow usage of keyword as positional, but that is also a breaking change for those who use keyword as positional.

Hope I am not overcomplicating this...

Reply:
Just making sure I understand...
It's the difference between `def load_data(directory, spec1d, spec2d, ...)` and `def load_data(spec1d=None, spec2d=None, *args)` where the latter needs to be referenced with `load_data(directory=dir_path)`, is that correct?

In that case, I would prefer users have all of their data in one directory so that we can do the former (directory first), but if they do not (such as the current data in MosvizExample.ipynb, which has images in a `cutout` directory inside the main directory), the latter would make more sense.